Add util getAxisValues in h5web/lib (#1123)
silx-kit/h5web: apps/storybook/src/Utilities.stories.mdx

scaleWithMid(3); // Returns 3
```

### getAxisValues

> Return a array of numbers with the desired axis length.

Reviewer: ```suggestion
Return an array of axis values of the desired length.
```
